pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -657,15 +674,21 @@ var _ = Describe("Converter", func() {
 			}
 		})
 
-		It("should use virtio-transitional models if requested", func() {
+		DescribeTable("should use virtio-transitional models if requested", func(arch string) {
 			v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
 			vmi.Spec.Domain.Devices.DisableHotplug = false
 			c.UseVirtioTransitional = true
+			c.Architecture = NewArchConverter(arch)
 			vmi.Spec.Domain.Devices.UseVirtioTransitional = &c.UseVirtioTransitional
 			dom := vmiToDomain(vmi, c)
 			testutils.ExpectVirtioTransitionalOnly(&dom.Spec)
-		})
+		},
+			Entry("on amd64 with success", "amd64"),
+			Entry("on arm64 with success", "arm64"),
+			Entry("on ppc64le with success", "ppc64le"),
+			//TODO add s390x entry with custom check of model used (disks/interfaces/controllers/devices will use different models)
```

Comment:
Is there a tracker for this todo as well?

Reply:
Nope

Reply:
Ok, i don't know what the general consensus on todos in code is, but from my perspective i would prefer you either implement it or at least open a tracker issue and reference it here.
In my experience todo comments in code is just far to easy to forget about.